evap/staff/views.py

```diff
@@ -783,6 +784,18 @@ def questionnaire_edit(request, questionnaire_id):
         return render(request, "staff_questionnaire_form.html", template_data)
 
 
+def get_identical_form_and_formset(questionnaire_id):
+    """
+    Generates a Questionnaire creation form and formset filled out like the already exisiting Questionnaire
+    specified in questionnaire_id. Used for copying and creating of new versions.
+    """
+    questionnaire = get_object_or_404(Questionnaire, id=questionnaire_id)
```

Comment:
please pull the `get_object_or_404` out of the method again and put it at the beginning of the two views